### ISSUE TITLE
switchInput breaks when used with shinydashBoard

### ISSUE THREAD

**Issue by andyquinterom** [dreamRs/shinyWidgets#462]
In version 0.6.3 the switchInput widget breaks with shinyDashboard. I'm guessing its an issue with bslib implentation and the boostrap libraries used in shinydashboard. However i am not sure if this should be fixed in here or over on the shinydashboard side.

**Comment by pvictor** [comment]
I don't think bslib is used in shinydashboard.
The example below works fine for me, can you share more infos ?

```r
library(shiny)
library(shinydashboard)
library(shinyWidgets)

shinyApp(
  ui = dashboardPage(
    dashboardHeader(),
    dashboardSidebar(),
    dashboardBody(
      switchInput(inputId = "somevalue"),
      verbatimTextOutput("value")
    ),
    title = "Dashboard example"
  ),
  server = function(input, output) {
    output$value <- renderPrint({ input$somevalue })
  }
)
```

**Comment by andyquinterom** [comment]
The error seems to be a little more fundamental. Initially I though it was about shinydashboard due to me having difficulty replicating the error. But it all seems to be caused by a single argument: `value`.

Test this out!

```R
library(shiny)
library(shinyWidgets)

shinyApp(
  ui = fluidPage(
    switchInput(
      inputId = "somevalue",
      value = TRUE
    ),
    verbatimTextOutput("value")
  ),
  server = function(input, output) {
    output$value <- renderPrint({ input$somevalue })
  }
)
```

Even tho the value is TRUE, the button is off.

**Comment by andyquinterom** [comment]
The problem gets fixed when dependencies are reverted to 3.3.4. I don't know if this breaks the bslib implementation, I would believe not.
